doc/source/methods/Anchors.ipynb

```diff
@@ -64,7 +64,9 @@
     "\n",
```

Comment:
<p>The <code>unknow/similarity</code>  doesn't show up that nice on the rendered docs, can we write it a different way? Also, I think we should merge the spacy + unknown/simiarity sampling sections to make it clear it's part of the same approach and also mention that the spacy stuff is only one choice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/432/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>merged them</p>